src/app/modules/candidate-registration/components/candidate-list/candidate-list.component.ts

```diff
@@ -0,0 +1,119 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { ToastrService } from 'ngx-toastr';
+import { RegistrationForm } from '../../modals/candidate';
+import { CandidateRegistrationService } from '../../services/candidate-registration.service';
+
+const candidate: RegistrationForm[] = [
+  {
+    id: '1',
+    name: 'Aakansha',
+    email: 'aakansha.choudhary@gmail.com',
+    techSkills: [
+      {
+        id: 0,
+        name: 'HTML',
+      },
+      {
+        id: 1,
+        name: 'CSS',
+      },
+    ],
+    techExperience: '5-7yrs',
+    openPosition: 'front End Developer',
+    completionDate: 'null',
+    result: 'null',
+  },
+  {
+    id: '1',
```

Comment:
If possible, update Id, name, email as all records looks similar

Reply:
once integrated we will be removing it, just for the display I added,